packages/client/src/sync/fetcher/fetcher.ts

```diff
@@ -352,10 +347,13 @@ export abstract class Fetcher<JobTask, JobResult, StorageItem> extends Readable
       return false
     }
     const jobStr = this.jobStr(job)
-    if (this._readableState === undefined || this._readableState!.length > this.maxQueue) {
+    if (
+      (<any>this)._readableState === undefined ||
+      (<any>this)._readableState!.length > this.maxQueue
+    ) {
       this.DEBUG &&
         this.debug(
-          `Readable state length=${this._readableState!.length} exceeds max queue size=${
+          `Readable state length=${(<any>this)._readableState!.length} exceeds max queue size=${
```

Comment:
This will not work if `_readableState` is undefined, and if we cast as `!` it will definitely error :thinking: